model/DeliveryResult/Normalizer/DeliveryResultNormalizer.php

```diff
@@ -54,29 +53,23 @@ public function normalize($deliveryExecution): IndexResource
         $deliveryExecutionId = $deliveryExecution->getIdentifier();
         $user = UserHelper::getUser($deliveryExecution->getUserIdentifier());
 
-        $customFieldService = $this->getResultCustomFieldsService();
-        $customBody = $customFieldService->getCustomFields($deliveryExecution);
-
         return new IndexResource(
             $deliveryExecutionId,
             $deliveryExecution->getLabel(),
-            array_merge(
-                [
-                    'label' => $deliveryExecution->getLabel(),
-                    self::INDEX_DELIVERY => $deliveryExecution->getDelivery()->getUri(),
-                    'type' => ResultService::DELIVERY_RESULT_CLASS_URI,
-                    self::INDEX_TEST_TAKER => $user->getIdentifier(),
-                    self::INDEX_TEST_TAKER_FIRST_NAME => UserHelper::getUserFirstName($user, true),
-                    self::INDEX_TEST_TAKER_LAST_NAME => UserHelper::getUserLastName($user, true),
-                    self::INDEX_TEST_TAKER_NAME => UserHelper::getUserName($user, true),
-                    self::INDEX_TEST_TAKER_LABEL => UserHelper::getUserLabel($user),
-                    self::INDEX_DELIVERY_EXECUTION => $deliveryExecutionId,
-                    self::INDEX_DELIVERY_EXECUTION_START_TIME => $this->transformDateTime(
-                        $deliveryExecution->getStartTime()
-                    )
-                ],
-                $customBody
-            )
+            [
+                'label' => $deliveryExecution->getLabel(),
+                self::INDEX_DELIVERY => $deliveryExecution->getDelivery()->getUri(),
+                'type' => ResultService::DELIVERY_RESULT_CLASS_URI,
+                self::INDEX_TEST_TAKER => $user->getIdentifier(),
+                self::INDEX_TEST_TAKER_FIRST_NAME => UserHelper::getUserFirstName($user, true),
+                self::INDEX_TEST_TAKER_LAST_NAME => UserHelper::getUserLastName($user, true),
+                self::INDEX_TEST_TAKER_NAME => UserHelper::getUserName($user, true),
+                self::INDEX_TEST_TAKER_LABEL => UserHelper::getUserLabel($user),
+                self::INDEX_DELIVERY_EXECUTION => $deliveryExecutionId,
+                self::INDEX_DELIVERY_EXECUTION_START_TIME => $this->transformDateTime(
+                    $deliveryExecution->getStartTime()
+                )
+            ]
```

Comment:
Removed because https://github.com/oat-sa/extension-tao-outcomeui/blob/master/model/search/ResultCustomFieldsService.php#L40 returns empty array

Reply:
It's not a case anymore